### PR TITLE
pm_runtime: longer hp sram power gating delay

### PR DESCRIPTION
**Pull request by slawblauciak**
Signed-off-by: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>

### CODE REVIEW

**Comment by ranj063** [comment]
@slawblauciak for my education how exactly does increasing the delay help? Do we understand it?

File: src/platform/intel/cavs/include/cavs/lib/pm_memory.h [hunk 1]
```diff
@@ -25,7 +25,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#define MEMORY_POWER_DOWN_DELAY 256
```
Comment (plbossart) [inline src/platform/intel/cavs/include/cavs/lib/pm_memory.h:28]: beyond clarifying why this needs to be a power of 2, adding a comment on the unit would help

**Comment by lgirdwood** [comment]
@slawblauciak ping
@ranj063 IIUC this is fixing the problem for you ?

**Comment by sys-pt1s** [comment]
Can one of the admins verify this patch?

**Comment by kv2019i** [comment]
No activity in a year, closing.

**Pull request closed without merging**